manifests/dbfile.pp

```diff
@@ -48,9 +50,9 @@
   } else {
 
     exec { "${postmap} ${title}":
-      cwd         => $postfixdir,
-      subscribe   => File["${postfixdir}/${title}"],
-      refreshonly => true,
+      cwd       => $postfixdir,
+      subscribe => File["${postfixdir}/${title}"],
+      creates   => "${postfixdir}/${title}.db".
```

Comment:
This fixes issue described on https://github.com/thias/puppet-postfix/issues/113